lib/linguist/languages.yml

```diff
@@ -1819,8 +1819,9 @@ Erlang:
   type: programming
   color: "#B83998"
   extensions:
-  - ".erl"
+  - ".app"
```

Comment:
You don't want to do this 😉 

The only time the alphabetisation doesn't apply is for the first extension. This is the primary extension for the language and I doubt it's changed from `.erl` to `.app` 😉

Reply:
Ah, sorry, I seem to have missed exactly the sentence about the primary extension in the guide.

Reply:
Top of the file:

https://github.com/github/linguist/blob/4101b6b6c118c0d978b1f4cd4c5bc66bd81a82d3/lib/linguist/languages.yml#L17-L19

Reply:
I know, was not contradicting you. It is also mentioned in the contribution guide.